rustls/src/client/client_conn.rs

```diff
@@ -186,12 +181,23 @@ pub struct ClientConfig {
     pub enable_early_data: bool,
 }
 
+/// What mechanisms to support for resuming a TLS 1.2 session.
+#[derive(Clone, Copy, Debug, PartialEq)]
+pub enum Tls12Resumption {
+    /// Disable 1.2 resumption.
+    Disabled,
+    /// Support 1.2 resumption using session ids only.
+    SessionIdOnly,
+    /// Support 1.2 resumption using session ids or RFC 5077 tickets.
```

Comment:
```suggestion
    /// Support 1.2 resumption using session ids or RFC 5077 tickets.
    ///
    /// See[1] for why you might like to disable RFC5077 by instead choosing the `SessionIdOnly` option.
    /// Note that TLS1.3 tickets do not have those issues.
    /// [^1]: https://words.filippo.io/we-need-to-talk-about-session-tickets/
```
(writing the markdown blind, please check)

Reply:
Folded into 55e1c6b7f1d2727554c7ef7978934245ce9de604 with a rebase. I tweaked the markdown slightly and verified it rendered correctly in `cargo doc --all-features`.